lib/components/Alert.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+
+/**
+ * Provides one of four standard alert types.
+ * An alert consists of a box with a light background color,
+ * an icon, a headline, and a short explaination.
+ *
+ * The alert type determines the background color and icon displayed
+ * 
+ * Alert types are 'success', 'warning', 'error', 'info'
+ * The type defaults to 'info' if no type is sent via props
+ *
+ * If the type is 'error' and no role is specified, role defaults to 'alert'
+ *
+ * @param {string} type  Sets the alert type
+ * @param {string, required} title Text for the headline
+ * @param {ndoe, required} body  Text for the description
+ * @param {string} role  ARIA role type
+ */
+export default function Alert ({type, title, body, role}) {
```

Comment:
You make a fine point.  I agree that it does make it super obvious that they are stateless.  If we're going to mix them, and you've sold me that it could be a good idea.  We should have some documentation in place on how/when to choose one over the other.

And I'm glad to hear that propTypes are still enforced.  I wasn't aware that they were.  I learned something new today!